src/components/migrationToV3.stories.mdx

```diff
@@ -20,97 +20,38 @@ npm install --save amazon-chime-sdk-component-library-react@beta amazon-chime-sd
-  const joinMeeting = async () => {
-    const response = await fetch('/my-meetings-endpoint');
-    const data = await response.json();
-    const joinData = {
-      meetingInfo: data.Meeting,
-      attendeeInfo: data.Attendee,
-      eventReporter: new NoOpEventReporter(),
-    };
-    try {
-      await meetingManager.join(joinData);
-    } catch {
-      console.error('Something went wrong');
-    }
-  };
-
-  return (
-    <button onClick={joinMeeting}>Join Meeting</button>;
-  );
-};
-```
-
-After in 3.x:
-
-```jsx
-const MyApp = () => {
-  const meetingManager = useMeetingManager();
-  const joinMeeting = async () => {
-    const response = await fetch('/my-meetings-endpoint');
-    const data = await response.json();
-    const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
-    const eventController = new DefaultEventController(
-      meetingSessionConfiguration, 
-      new ConsoleLogger('SDK', LogLevel.WARN), 
-      new NoOpEventReporter()
-    );
-    const options = {
-      eventController
-    };
-
-    try {
-      await meetingManager.join(
-        meetingSessionConfiguration,
-        options
-      );
-    } catch {
-      console.error('Something went wrong');
-    }
-  };
-  
-  return (
-    <button onClick={joinMeeting}>Join Meeting</button>;
-  );
-};
-```
+- Remove `meetingRegion` from `MeetingManager`. Previously we just cache this value from meetingInfo returned by
+[CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html) but it is not used
+anywhere in the library.
+- Deprecate `useBandwidthMetrics` hook in favor of `useMediaStreamMetrics`.
+- Remove legacy metrics `videoDownstreamGoogFrameHeight`, `videoDownstreamGoogFrameWidth`, `videoUpstreamGoogFrameHeight` and `videoUpstreamGoogFrameWidth` from the `videoStreamMetrics` returned by the `useMediaStreamMetrics` hook to adopt to Amazon Chime SDK for JavaScript V3 changes ([aws/amazon-chime-sdk-js#2086](https://github.com/aws/amazon-chime-sdk-js/pull/2086)).
+- Update the `compilerOptions.target` in `tsconfig.json` from `es5` to `ES2015 (ES6)`.
+- Rename the `global` property of `DefaultTheme` Interface to `globalStyle` to avoid conflict with reserved keyword `global`.
 
 ### `MeetingProvider` props change
 
 `MeetingProvider` no longer takes the following props:
 - `MeetingSessionConfiguration` properties:
-  - `VideoDownlinkBandwidthPolicy`
-  - `VideoUplinkBandwidthPolicy`
-  - `simulcastEnabled`
-  - `reconnectTimeoutMs`
-  - `keepLastFrameWhenPaused`
+- `VideoDownlinkBandwidthPolicy`
+- `VideoUplinkBandwidthPolicy`
+- `simulcastEnabled`
+- `reconnectTimeoutMs`
+- `keepLastFrameWhenPaused`
 - Other props:
-  - `deviceLabels`
-  - `eventController`
-  - `logLevel`
-  - `postLogConfig`
-  - `logger`
-  - `enableWebAudio`
-  - `activeSpeakerPolicy`
+- `deviceLabels`
+- `eventController`
+- `logLevel`
+- `postLogConfig`
+- `logger`
+- `enableWebAudio`
+- `activeSpeakerPolicy`
```

Comment:
Why we remove the indention? I think we should either remove dash for "MeetingSessionConfiguration properties:" and "Other props:" or add intention to their children bullet points.

Reply:
It looks like this now:

- `MeetingSessionConfiguration` properties:
- `VideoDownlinkBandwidthPolicy`
- `VideoUplinkBandwidthPolicy`
- `simulcastEnabled`
- `reconnectTimeoutMs`
- `keepLastFrameWhenPaused`
- Other props:
- `deviceLabels`
- `eventController`
- `logLevel`
- `postLogConfig`
- `logger`
- `enableWebAudio`
- `activeSpeakerPolicy`